modules/couchdb/collect.go

```diff
@@ -80,11 +82,23 @@ func (CouchDB) collectActiveTasks(collected map[string]int64, ms *cdbMetrics) {
 	}
 }
 
-func (CouchDB) collectDBStats(collected map[string]int64, ms *cdbMetrics) {
+func (cdb *CouchDB) collectDBStats(collected map[string]int64, ms *cdbMetrics) {
 	if !ms.hasDBStats() {
 		return
 	}
 
+	for _, dbStats := range ms.DBStats {
+		if dbStats.Error != "" {
+			cdb.Error("database '", dbStats.Key, "' doesn't exist")
+			continue
+		}
+		metrics := make(map[string]int64)
+		renameKeys(metrics, stm.ToMap(dbStats.Info), "db_"+dbStats.Key)
+		for metric, value := range metrics {
+			collected[metric] = value
+		}
```

Comment:
i think it is just 

```go
prefix := ...
for k, v := range stm.ToMap(dbStats.Info) {
	collected[prefix + "_" + k] = value
}
```

Reply:
or if you would use `merge`

`merge(collected, stm.ToMap(dbStats.Info), "db_"+dbStats.Key)`